src/domain/entities/Settings.h

```diff
@@ -33,6 +35,10 @@ extern "C"
 
             String getDeviceName();
             void setDeviceName(String deviceName);
+            String getWifiSSID();
+            void setWifiSSID(String wifiSSID);
+            String getWifiPassword();
+            void setWifiPassword(String wifiPassword);
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.